### DISCUSSION TITLE
Change Failure Rate: issues and interval

### DISCUSSION THREAD

**Discussion by weshayutin** [dora-metrics/pelorus#671] [Q&A]
Once a production bug is created afaict it is always accounted for regardless of the interval.
It's not 100% clear to me that `change failure rate` should be different in the other three metrics in this way.

* Should we ensure that CFR only count bugs opened in the the interval?
* If keeping the total count of bugs should be part of the formula, should we allow customers to label some bugs so that they are no longer counted? escape hatch.  This could lead to score manipulation.

Perhaps we just have a bug in the following or maybe I'm just misunderstanding how this works:
Change Failure Rate over time
	avg_over_time(sdp:change_failure_rate:global [$__range])

Thoughts

**Comment by KevinMGranger** [comment]
The rate is supposed to be for an interval, though, right? That's how you can say "a month ago our failure rate was X, and after adopting these changes this month it's Y"
